webonise5/src/services/StringOperationsService.java

```diff
@@ -0,0 +1,20 @@
+package services;
+
+import java.util.Iterator;
+import java.util.LinkedHashSet;
+import java.util.Scanner;
+
+public class StringOperationsService {
+
+   public void removeDuplicates(String stringToRemoveDuplicates) {
+        LinkedHashSet<String> inputSet = new LinkedHashSet<>();
+        for(String stringSlice : stringToRemoveDuplicates.split(" ")){
+            inputSet.add(stringSlice);
+        }
+        stringToRemoveDuplicates = "";
+        for(String itemFromInputSet : inputSet) {
+            stringToRemoveDuplicates = stringToRemoveDuplicates + " " + itemFromInputSet;
```

Comment:
could have used single for loop for removing duplicates, just think over it once.